task19/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
Q.
I thought it was unnecessary to mention this since css/style.css is loaded on the JS side. Please let me know if I am thinking differently!

Reply:
You are also right!I don't need to read it in.
https://github.com/seeeen044/JavaScript_handson/commit/b4d7d86b721fa44804f102498636163c7181f778